### ISSUE TITLE
<Insert summary of task>

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#14898]
## User Story or Problem Statement

Review implementation in #14701 for accessibility

Refer to [VBA Content Specs XLS in Sharepoint](https://dvagov.sharepoint.com/:x:/s/SitewideFacilitiesTeam/EVVWpQp091ZEvI2n-9iyvTQBGYFZjxlVPN3Zfyz-Eh6Oyg?e=NVvsCc) -- Legend tab explains color highlighting

## Tasks
- [ ] Review editorial interface components for accessibility
- [ ] Review all text (help text, field labels, tool tips, error messages) for accessibility best practices
- [ ] Consider broader user flow(s) for the task(s) and make sure this logically integrates/references other Drupal entities
- [ ] Consult with Engineering to understand what technical limitations may prevent updating interface
- [ ] Document any recommendations for improving the interface

## Acceptance Criteria
- [ ] Recommendations are documented
- [ ] Recommendations are shared with PM/PO
## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [X] Facilities
- [ ] User support
- [ ] Accelerated Publishing

**Comment by xiongjaneg** [comment]
@swirtSJW Would you please add a comment here or in the description for how Laura can get to the content type to review?

**Comment by davidmpickett** [comment]
While this is not fully blocked by #14700, it might make sense to wait for that since it will be updating this content type

**Comment by laflannery** [comment]
### Comments/Questions

1. Can we update the "Jump to edit" links?
   1. Remove Services: This link doesn't work and I also can't see where it would go so I don't think we need this.
   2. To be consistent, I would like to add links for:
      1. Banner alert (after Operating status)
      2. Prepare for your visit
      3. Spotlight content
2. Also, I'm confused on the Facility data section. The whole section has a dark gray background and a tool tip that indicates that I can't edit these fields. However, only some fields are actually read-only. The majority are fields I can actually type into and change the filled in data (though I didn't try to save the changed data because this was on Prod - maybe upon saving it would stop me?).
   1. I feel that this is a confusing experience - if this information actually cannot be edited, can we make this truly not editable? 
3. The help text for the Facility Photo is not fabulous. This is also identical to Vet Center's right now. Is it within the scope of this work to update/change the VBA facility photo help text or is that outside the scope because it would change Vet Center content types as well? (To be clear I wouldn't necessarily want the help text different for VBA and Vet Center, I do want it changed for both.)

**Comment by davidmpickett** [comment]
> 2. Also, I'm confused on the Facility data section. The whole section has a dark gray background and a tool tip that indicates that I can't edit these fields. However, only some fields are actually read-only. The majority are fields I can actually type into and change the filled in data (though I didn't try to save the changed data because this was on Prod - maybe upon saving it would stop me?).
> I feel that this is a confusing experience - if this information actually cannot be edited, can we make this truly not editable?

Were you logged in as yourself when reviewing this? These fields are locked down for Content Editors & Reviewers & Publishers, but editable for Content Admins and True Admins. When I log in as Tanekwa.Bournes@va.gov, this whole section is grayed out.

![screencapture-training-qtpbfivdfzrlrlyiqwmanznaobpek1ca-demo-cms-va-gov-node-4338-edit-2023-11-27-11_47_58](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/50370185-1572-497e-bb60-c0b416c6fd0a)

That being said, we are breaking an existing pattern here since this data will truly be controlled in Drupal as opposed to migrated in from VAST, so we're going to need clear documentation on who/how this information gets updated. Future tickets to address some of these things:

- #15396
- #15366
- #15386

**Comment by davidmpickett** [comment]
> 3. The help text for the Facility Photo is not fabulous. This is also identical to Vet Center's right now. Is it within the scope of this work to update/change the VBA facility photo help text or is that outside the scope because it would change Vet Center content types as well? (To be clear I wouldn't necessarily want the help text different for VBA and Vet Center, I do want it changed for both.)

The following help text is within scope. Whatever we put here will be specific to this Content Type (though I agree that any edits here should also be reflected in other places this field is used)

```
<strong>Photo guidelines</strong> <br>
✓ Smart phone photos are ok <br>
✓ Use landscape orientation <br>
✓ Show the entrance side of the building <br>
✗ No people <br>
✗ No car license plates<br>
```

**Comment by laflannery** [comment]
@xiongjaneg Can you take a look at the comments above and let me know what's actionable/in scope from these comments and if there are any additional tickets I should make? To summarize [the original comment](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14898#issuecomment-1828296919):
1. Still pending (possibly needs an engineer?)
2. I was logging in as the wrong editor, I withdraw the question :smile:
3. Still pending whether or not this is in scope and would require a separate ticket

**Comment by xiongjaneg** [comment]
@laflannery 
1. Have you added these comments to the Figma file? If so, I can spin up tickets for updates to design and subsequent Drupal.
3. I think this it's best to go forward with your recommendations. Again, are these comments added to the Figma file? If so, I can include them in the design and subsequent Drupal ticket. I could also create a change management ticket for Vet Centers which will likely be a KB update.

Does this give you what you need to finish this ticket?

**Comment by xiongjaneg** [comment]
@laflannery Perfect, thank you!

**Comment by thejordanwood** [comment]
@laflannery I've created [the Figma doc](https://www.figma.com/file/aKdT4weNNthDxb3r6MolEh/VBA-Content-Specs?type=design&node-id=500%3A154&mode=design&t=SpLKfoqJ8nUVMYM8-1) for this now!

I’ve included some of the changes you brought up in this ticket, but am interested to hear if you have any thoughts on what the Facility photo help text should change to.
